src/main/resources/liquibase/crop_changelog/16_3_0.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="diego" id="v16.3.0-1">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*)
+					FROM information_schema.routines
+						WHERE routine_schema = DATABASE() AND specific_name = 'deleteTermIdFromProjectPropTable';
+			</sqlCheck>
+		</preConditions>
+		<comment>Drop procedure deleteTermIdFromProjectPropTable</comment>
+		<sql dbms="mysql" splitStatements="false">
+			DROP PROCEDURE deleteTermIdFromProjectPropTable;
+		</sql>
+	</changeSet>
+
+	<changeSet author="diego" id="v16.3.0-2">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT COUNT(*)
+					FROM information_schema.routines
+						WHERE routine_schema = DATABASE() AND specific_name = 'deleteTermIdFromProjectPropTable';
+			</sqlCheck>
+		</preConditions>
+		<comment>Create procedure deleteTermIdFromProjectPropTable</comment>
+		<sql dbms="mysql" splitStatements="false">
+			CREATE PROCEDURE deleteTermIdFromProjectPropTable(termId int)
+			BEGIN
+				DECLARE projectId INT;
+				DECLARE done INT DEFAULT FALSE;
+				DECLARE cur1 CURSOR FOR SELECT project_Id FROM projectprop WHERE variable_id = 8269 order by rank;
+				DECLARE CONTINUE HANDLER FOR NOT FOUND SET done = TRUE;
+
+				OPEN cur1;
+					read_loop: LOOP FETCH cur1 INTO projectId;
+						set @newRank = 0;
+						IF done THEN
+							LEAVE read_loop;
+						END IF;
+
+						DELETE FROM projectprop WHERE project_id = projectId AND variable_id = termId;
```

Comment:
Did you run this procedure in a large database? 
I think that storing in a temporary table the rows that will be deleted, then executing only one DELETE statement before the loop and then loop the temporaty table to update rank can improve the performance